### DISCUSSION TITLE
Flutter SDK Version

### DISCUSSION THREAD

**Discussion by claireebilski** [codemagic-ci-cd/codemagic-docs#1684] [Q&A] [answered]
Hi there, 

I am using the Default Workflow for deploying my iOS flutter app and the flutter sdk version is defaulting to 3.0.1 when I want it to be 2.12.0+1... how do I specify this? I have looked at FVM and am not sure how the flavor is working. Do I have to download FVM somewhere? Can I use 2.12.0+1 as the flavor?

**Comment by dtrdic** [comment, marked as answer]
Hey @claireebilski, there is a Flutter Version Management (FVM) tool available that allows you to easily select the desired Flutter version specified in your FVM configuration file. You can download the FVM package from here https://fvm.app/docs/getting_started/overview/

Once you have FVM set up, you can use it in Codemagic by selecting the "from FVM configuration" option from the dropdown menu in the Build settings. This will ensure that your project is built using the Flutter version specified in your FVM configuration.

**Reply by claireebilski** [reply]
Hi @dtrdic, thank you for that response. 

I went through the process you described and got an error "FVM configuration file not found at /Users/builder/clone/.fvm/fvm_config.json" so I went ahead and tried manually setting the version in the Build settings and then my build completed...
